testutil/sample/sample_ton.go

```diff
@@ -18,16 +17,17 @@ import (
 )
 
 const (
-	tonWorkchainID    = 0
-	tonShardID        = 123
-	tonDepositFee     = 10_000_000 // 0.01 TON
-	tonSampleGasUsage = 50_000_000 // 0.05 TON
+	tonWorkchainID   = 0
+	tonShardID       = 123
+	tonSampleTXFee   = 006_500_000 // 0.0065 TON
```

Comment:
We generally use `Tx` not `TX` throughout the code, as it is not an acronym I think it makes more sense to stick to Tx